Assets/Scripts/WeaponAttach.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class WeaponAttach : MonoBehaviour
+{
+    public Sword sword;
+    public Axe axe;
+    public GameObject swordWeapon;
+    public GameObject axeWeapon;
+    public GameObject weaponHolder;
+    public EquippedWeapon currentWeapon;
+
+
+    private void Update()
+    {
+        if (Input.GetMouseButtonDown(1)) axeWeapon.transform.SetParent(weaponHolder.transform);
```

Comment:
I'm not quite sure I understand this, why do you need to update the parent every frame?

Why is it only the `axeWeapon` being mentioned here?